.eslintrc.js

```diff
@@ -3,9 +3,14 @@ module.exports = {
   parser: "@typescript-eslint/parser",
   plugins: ["@typescript-eslint"],
   extends: ["standard-with-typescript", "prettier"],
-  parserOptions: {
-    project: "tsconfig.json",
-  },
+  overrides: [
+    {
+      files: ["*.ts", "*.tsx"],
+      parserOptions: {
+        project: ["tsconfig.json"],
+      },
+    },
+  ],
```

Comment:
This fixes an issue (might be a VS Code thing) where .js files were being checked with the tsconfig settings via eslint (if that makes sense).

Reply:
I think your VSC is applying defaults. Bertrand and Mo were VSC-ers And I don't think this caused any issues. Is your editor pointing to the right file? or the right eslint binary?